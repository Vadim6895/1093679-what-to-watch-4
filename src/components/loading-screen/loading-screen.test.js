import React from "react";
import renderer from "react-test-renderer";
import LoadingScreen from "./loading-screen.jsx";

it(`Should ErrorScreen render correctly`, () => {
  const tree = renderer
  .create(
      <LoadingScreen/>)
  .toJSON();

  expect(tree).toMatchSnapshot();
});
