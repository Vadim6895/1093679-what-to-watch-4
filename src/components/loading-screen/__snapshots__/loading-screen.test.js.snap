// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should ErrorScreen render correctly 1`] = `
<div
  style={
    Object {
      "background": "rgb(0, 0, 0)",
      "height": "100vh",
      "width": "100%",
    }
  }
>
  <svg
    height="204px"
    preserveAspectRatio="xMidYMid"
    style={
      Object {
        "background": "rgb(0, 0, 0)",
        "display": "block",
        "margin": "auto",
        "paddingTop": "100px",
        "shapeRendering": "auto",
      }
    }
    viewBox="0 0 100 100"
    width="204px"
    xmlns="http://www.w3.org/2000/svg"
    xmlnsXlink="http://www.w3.org/1999/xlink"
  >
    <g>
      <path
        d="M50 15A35 35 0 1 0 74.74873734152916 25.251262658470843"
        fill="none"
        stroke="#0071bf"
        strokeWidth="12"
      />
      <path
        d="M49 0L49 30L64 15L49 0"
        fill="#0071bf"
      />
      <animateTransform
        attributeName="transform"
        dur="4.761904761904762s"
        keyTimes="0;1"
        repeatCount="indefinite"
        type="rotate"
        values="0 50 50;360 50 50"
      />
    </g>
  </svg>
</div>
`;
